src/Components/CommentDelete.jsx

```diff
@@ -15,12 +15,13 @@ export default function CommentDelete(props) {
     document.getElementById(`deletingLoading${comment_id}`).showModal();
     deleteCommentByArticleId(comment_id)
       .then(() => {
+        console.log("in the then block");
         document.getElementById(`deletingLoading${comment_id}`).close();
         document.getElementById(`commentDeleted${comment_id}`).showModal();
       })
       .catch(() => {
+        console.log("in the error block");
         document.getElementById(`deletingLoading${comment_id}`).close();
```

Comment:
is this the way to do this, i think you should be looking to use useRef to handle document type stuff as ract uses a virtual dom